test/test_build.py

```diff
@@ -39,6 +39,43 @@ class ImageBuildResult(NamedTuple):
     metadata: dict = {}
 
 
+def get_distribution_and_version(test_vm, build_ansible_container, username, password):
+    """Run Ansible playbook to get distribution and version"""
+
+    output = test_vm.execute_ansible(build_ansible_container,
+                                     "./test/ansible-playbooks/get_distro_version.yml",
+                                     username,
+                                     password)
+
+    print(output)
+
+    # Parse output to extract distribution and version
+    distribution = None
+    version = None
+    dist_key = "Distribution:"
+    version_key = "Version:"
+    for line in output.splitlines():
+        if dist_key in line:
+            distribution = line.split(dist_key)[1].strip(" \"'")
+        elif version_key in line:
+            version = line.split(version_key)[1].strip(" \"'")
+
+    return distribution, version
+
+
+def check_user(test_vm, build_ansible_container, username, password, user_to_test):
+    """Run Ansible playbook to get distribution and version"""
+
+    output = test_vm.execute_ansible(build_ansible_container,
```

Comment:
I know this is meant to grow, but right now it feels very heavy to build an ansible container, run a playbook, get the output and return the user. All of this is essentially: `test_vm.run(f"getent passwd {user_to_test}")`. A bit similar for the get_distro() it does essentially `test_vm.run("cat /etc/os-release")`